docs/source/install.rst

```diff
@@ -11,46 +11,61 @@ Development
 
    .. code-block:: bash
 
-      $ conda env create -f environment.yml
```

Comment:
FYI - I removed the $ as I found it annoying that I would always get the `$` when I copied the compile protos command. There might be a cleaner way of ensuring this?